### ISSUE TITLE
fix yaml loader usage

### ISSUE THREAD

**Issue by sbailey** [desihub/desispec#783]
From the unit tests:
```
/Users/sbailey/desi/git/desispec/py/desispec/maskbits.py:73: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
```
We fixed this in one of our other repos (I forget which one); go find which one and what we did there and do the same in desispec.

**Comment by geordie666** [comment]
We did this in `desitarget`:

https://github.com/desihub/desitarget/blob/master/py/desitarget/targetmask.py#L22

**Comment by sbailey** [comment]
Fixed in PR #801
